src/langchainWrappers.ts

```diff
@@ -24,6 +27,72 @@ export class ProxyOpenAIEmbeddings extends OpenAIEmbeddings {
     this["client"] = new OpenAI({
       ...this["clientConfig"],
       baseURL: fields.openAIEmbeddingProxyBaseUrl,
+      fetch: fields.useOpenAILocalProxy ? safeFetch : undefined,
```

Comment:
Since embedding endpoints don't need streaming, perhaps we should default to `safeFetch`? WDYT?

Reply:
Yes, it's not only using `safeFetch()`, I tested it adding `openAIEmbeddingProxyBaseUrl = 'http://localhost:1234/v1'` and using LMStudio with LLama 3 and it's re-endexing all the embeddings when I force re-index.